src/mixer/basetrackplayer.h

```diff
@@ -162,6 +166,8 @@ class BaseTrackPlayerImpl : public BaseTrackPlayer {
 
     // Track color control
     std::unique_ptr<ControlObject> m_pTrackColor;
+    std::unique_ptr<ControlPushButton> m_pTrackColorPrev;
+    std::unique_ptr<ControlPushButton> m_pTrackColorNext;
```

Comment:
While you're at it:
I think a `track_color_selector` will also be helpful, that could be mapped to rotary encoders right away with the MIDI Wizard.

Reply:
Btw, please add the new controls to controlpickermenu.cpp, maybe after stars?
https://github.com/mixxxdj/mixxx/blob/20de6fa9fad8b7e86401ae7651b2379adcec6254/src/controllers/controlpickermenu.cpp#L1388

Reply:
> While you're at it: I think a `track_color_selector` will also be helpful, that could be mapped to rotary encoders right away with the MIDI Wizard.

Done

Reply:
Done for controlpickermenu.cpp too